x-pack/filebeat/module/zeek/mqtt_connect/_meta/fields.yml

```diff
@@ -0,0 +1,20 @@
+- name: mqtt_connect
+  type: group
+  default_field: false
+  description: >
+    Fields exported by the Zeek mqtt_connect log.
+  fields:
+    - name: proto_version
+      type: keyword
+      description: >
+        The version of the protocol in use.
+
+    - name: client_id
+      type: keyword
+      description: >
+        Unique identifier for the client.
+
+    - name: connect_status
+      type: keyword
+      description: >
+        Status message from the server in response to the connect request.
```

Comment:
Looking at https://docs.zeek.org/en/master/scripts/policy/protocols/mqtt/main.zeek.html, it looks like MQTT::ConnectInfo can have some other fields not listed like will_topic & will_payload.  I think we should add those so we don't have a default mapping (if it is possible for them to appear in the logs).

Reply:
Yes, if are documented probably these fields can appear. so better to add. At that time when I tested, these fields didn't appear.